notebooks/01_collect_data/raw_to_text.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<p>I don't think you need to print the "subset_mboxes" here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>